Scripts/Editor/Configuration/WitConfigurationEditor.cs

```diff
@@ -328,14 +338,22 @@ private void DrawApplication(WitApplication application)
         }
         else
         {
-            InfoField("Name", application.name);
-            InfoField("ID", application.id);
-            InfoField("Language", application.lang);
-            InfoField("Created", application.createdAt);
-            GUILayout.BeginHorizontal();
-            GUILayout.Label("Private", GUILayout.Width(100));
-            GUILayout.Toggle(application.isPrivate, "");
-            GUILayout.EndHorizontal();
+            if (!string.IsNullOrEmpty(WitAuthUtility.AppServerToken) && WitAuthUtility.AppServerToken.Length == 32)
```

Comment:
it's a bit weird to have this here
what do you think @yolanother?

Reply:
Yeah this doesn't feel right. We should probably use a custom property drawer to display application info so we can then override that instead.

Reply:
addressed